packages/@aws-cdk/aws-efs/test/test.efs.ts

```diff
@@ -1,8 +1,172 @@
-import { Test, testCase } from 'nodeunit';
+import {App, Stack} from '@aws-cdk/cdk';
```

Comment:
1. Please use spaces inside `{}`
2. Please use @aws-cdk/assert for finer grained assertions and no need to implement `TestApp`.

```ts
import { expect, haveResource } from '@aws-cdk/assert';

export = {
  'default properties': {
    const stack = new cdk.Stack();
    new FileSystem(stack);
    expect(stack).to(haveResource('AWS::EFS::FileSystem', {
      Encrypted: true,
      PerformanceMode: 'generalPurpose',
      ProvisionedThroughputInMibps: 100,
      ThroughputMode: 'bursting'
    }));
    test.done();
  }
}
```

Reply:
ohh, I see now. I misunderstood your last comment. I'll add the spaces.

Reply:
I didn't know that existed. That makes these tests much simpler. thanks.